lib/fpm/package/virtualenv.rb

```diff
@@ -23,6 +23,12 @@ class FPM::Package::Virtualenv < FPM::Package
     File.expand_path(path)
   end
 
+  option "--bin", "VIRTUALENV_EXECUTABLE",
+  "The path to the virtualenv executable you wish to run.", :default => "virtualenv"
+  
+  option "--python", "VIRTUALENV_PYTHON",
+  "The version of python to create the virtualenv with", :default => "python"
```

Comment:
I wonder if this should read "The path to the python binary to use with virtualenv" ? `python` doesn't sound like a version to me.

That said, I don't think we need to block the PR on this.

Reply:
```suggestion
  "The path to the python binary to use with virtualenv", :default => "python"
```